docs/source/conf.py

```diff
@@ -291,344 +283,16 @@
-
-
-git_branch = _git_branch()
-base_url = "https://github.com/python-telegram-bot/python-telegram-bot/blob/"
-
-
-def linkcode_resolve(_, info):
-    """See www.sphinx-doc.org/en/master/usage/extensions/linkcode.html"""
-    combined = ".".join((info["module"], info["fullname"]))
-    # special casing for ExtBot which is due to the special structure of extbot.rst
-    combined = combined.replace("ExtBot.ExtBot", "ExtBot")
-
-    line_info = line_numbers.get(combined)
-
-    if not line_info:
-        # Try the __init__
-        line_info = line_numbers.get(f"{combined.rsplit('.', 1)[0]}.__init__")
-    if not line_info:
-        # Try the class
-        line_info = line_numbers.get(f"{combined.rsplit('.', 1)[0]}")
-    if not line_info:
-        # Try the module
-        line_info = line_numbers.get(info["module"])
-
-    if not line_info:
-        return
-
-    file, start_line, end_line = line_info
-    return f"{base_url}{git_branch}/{file}#L{start_line}-L{end_line}"
-
-
-# End of logic for the [source] links
-# ------------------------------------------------------------------------------------------------
-
-
-# Some base classes are implementation detail
-# We want to instead show *their* base class
-PRIVATE_BASE_CLASSES = {
-    "_ChatUserBaseFilter": "MessageFilter",
-    "_Dice": "MessageFilter",
-    "_BaseThumbedMedium": "TelegramObject",
-    "_BaseMedium": "TelegramObject",
-    "_CredentialsBase": "TelegramObject",
-}
-
-
-def autodoc_process_bases(app, name, obj, option, bases: list):
-    """Here we fine tune how the base class's classes are displayed."""
-    for idx, base in enumerate(bases):
-        # let's use a string representation of the object
-        base = str(base)
-
-        # Special case because base classes are in std lib:
-        if "StringEnum" in base == "<enum 'StringEnum'>":
-            bases[idx] = ":class:`enum.Enum`"
-            bases.insert(0, ":class:`str`")
-            continue
-
-        if "IntEnum" in base:
-            bases[idx] = ":class:`enum.IntEnum`"
-            continue
-
-        # Drop generics (at least for now)
-        if base.endswith("]"):
-            base = base.split("[", maxsplit=1)[0]
-            bases[idx] = f":class:`{base}`"
-
-        # Now convert `telegram._message.Message` to `telegram.Message` etc
-        match = re.search(pattern=r"(telegram(\.ext|))\.[_\w\.]+", string=base)
-        if not match or "_utils" in base:
-            continue
-
-        parts = match.group(0).split(".")
-
-        # Remove private paths
-        for index, part in enumerate(parts):
-            if part.startswith("_"):
-                parts = parts[:index] + parts[-1:]
-                break
-
-        # Replace private base classes with their respective parent
-        parts = [PRIVATE_BASE_CLASSES.get(part, part) for part in parts]
-
-        base = ".".join(parts)
 
-        bases[idx] = f":class:`{base}`"
+# Not used but must be imported for the linkcode extension to find it
```

Comment:
maybe also add a comment that the sphinx logger was imported in the other files in docs.auxil so not needed to reimport

Reply:
What's benefit in mentioning that here? If we don't need a logger here anyway, why comment on it ... :)